octobot_evaluators/evaluators/evaluator_factory.py

```diff
@@ -167,3 +158,31 @@ async def create_all_type_evaluators(tentacles_setup_config: object,
     except ImportError:
         logging.get_logger(LOGGER_NAME).error("create_evaluators requires Octobot-Trading package installed")
     return []
+
+
+def _extract_traded_pairs(symbols_by_crypto_currencies, exchange_name, matrix_id):
+    crypto_currency_name_by_crypto_currencies = {}
+    symbols_by_crypto_currency_tickers = {}
+    import octobot_trading.api as exchange_api
+    exchange_id = exchange_api.get_exchange_id_from_matrix_id(exchange_name, matrix_id)
+    exchange_manager = exchange_api.get_exchange_manager_from_exchange_name_and_id(exchange_name, exchange_id)
+    for name, symbol_list in symbols_by_crypto_currencies.items():
+        if symbol_list:
+            # in case pairs are listed by reference market in use config, iterate over each pair
+            for symbol in symbol_list:
+                base = exchange_api.get_base_currency(exchange_manager, symbol)
+                crypto_currency_name_by_crypto_currencies[base] = \
+                    crypto_currency_name_by_crypto_currencies.get(base, name)
```

Comment:
It was `crypto_currency_name_by_crypto_currencies.get(base, "") + name` and now `crypto_currency_name_by_crypto_currencies.get(base, name)` ?

Reply:
yep, it was an issue

Reply:
:+1: